network/endpoint_windows_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Microsoft. All rights reserved.
+// MIT License
+
+// +build !windows
+
+package network
+
+import (
+	"fmt"
+	"github.com/Azure/azure-container-networking/network/hnswrapper"
+	"net"
+	"testing"
+)
+
+func TestNewAndDeleteEndpointImplHnsV2(t *testing.T){
+	nw := &network{
+		Endpoints: map[string]*endpoint{},
+	}
+	hnsv2 = hnswrapper.Hnsv2wrapperFake{}
```

Comment:
Where is this hnsv2 being used ?

Reply:
This is package level, I do this to avoid passing around os specific code such as HNS in platform agnostic code, unfortunately its not the best but given our setup, I think its a better approach.